scripts/get_bank_registry_at.py

```diff
@@ -0,0 +1,33 @@
+import json
+import requests
+
+URL = "https://www.oenb.at/docroot/downloads_observ/sepa-zv-vz_gesamt.csv"
+
+
+def process():
+    registry = []
+    count = 0
+
+    with requests.get(URL, stream=True) as csvfile:
+        for row in csvfile.iter_lines():
```

Comment:
Instead of counting the rows I would suggest to use the slicing operator. Additionally I would use the `csv`-module for "parsing" the CSV file
```python
for row in csv.reader(list(csvfile.iter_lines())[6:]):
    ...
```